contrib/mhchem/mhchem-manual.html

```diff
@@ -0,0 +1,279 @@
+    h2 > strong { font-family: 'Alegreya',serif; text-transform: none; font-variant: inherit; font-size: 90%; }
+    h2 > small { font-size: 60%; display: block; word-spacing: 0.2em; margin-top: -0.5em; }
+    p { margin-top: 0.5rem; }
+    .example { margin: 1rem 2rem 1rem 0; display: inline-block; vertical-align: top; }
+    .a, .b, .c { text-align: center; }
+    .a { padding-left: 1.8rem; }
+    .b { color: rgba(163,62,0,1); padding: 0 .25rem; margin: 0.25rem 0 0 0; border-radius: 0.25rem; transition: background-color 3s; }
+    body:hover .b { background-color: #f0f0f0; transition: background-color 1s; }
+    code { font-family: 'Source Code Pro', monospace; font-size: 90%; white-space: pre-wrap; padding: 0 0.2rem; }
+    .b code.before, .b code.after { color: #aaa; }
+    .c { font-size: 80%; }
+    em { font-style: italic; }
+    a { color: #6666cc; text-decoration: none; }
+    a:hover { text-decoration: underline; }
+    ul { list-style: none; }
+    li { margin: 0.5rem 0; }
+    blockquote { margin-left: 2rem; font-size: 90%; }
+    #input { display: block; width: 100%; font-family: 'Source Code Pro', monospace; font-size: 95%; color: rgba(163,62,0,1); background-color: #ffffff; border: 1px solid #ccc; border-radius: 4px; padding: 0.3rem 0.6rem; }
+    #input:focus { background-color: #fff; }
+    #output { margin: 1rem; min-height: 2rem; }
+    .footer { margin-top: 2rem; text-align: center; color: #ccc; }
+    .hidden { display: none; }
+  </style>
+  <title>KaTeX/mhchem Manual</title>
+</head>
+<body>
+  <div id="md">Please ensure Internet access and enable JavaScript.</div>
+  <textarea id="ta" class="hidden">
+
+# KaTeX/mhchem Manual
+
+mhchem is a tool for writing beautiful chemical equations easily.
+
+This is the manual for mhchem's input syntax. It covers version 3.3 of KaTeX/mhchem.
+
+For information on how to load the extension and make the `\ce` command available in KaTeX, see the [official KaTeX docs](https://github.com/Khan/KaTeX/blob/master/contrib/mhchem/README.md).
+
+## Chemical Equations (ce)
+:::: CO2 + C -> 2 CO
+:::: Hg^2+ ->[I-] HgI2 ->[I-] [Hg^{II}I4]^2-
+::::$ C_p[\ce{H2O(l)}] = \pu{75.3 J // mol K}
+
+## Chemical Formulae
+:::: H2O
+:::: Sb2O3
+
+## Charges
+:::: H+
+:::: CrO4^2-
+:::: [AgCl2]-
+:::: Y^99+
+:::: Y^{99+}
+
+## Stoichiometric Numbers
+:::: 2 H2O
+:::: 2H2O
+:::: 0.5 H2O
+:::: 1/2 H2O
+:::: (1/2) H2O % IUPAC Green Book
+:::: $n$ H2O
+
+## Isotopes
+:::: ^{227}_{90}Th+
+:::: ^227_90Th+
+:::: ^{0}_{-1}n^{-}
+:::: ^0_-1n-
+It might be ambiguous whether a superscript belongs to the left or right element. There is automatic detection (digits only = mass number = belongs to right side), but to make sure you can type `{}` as a separator.
+:::: H{}^3HO
+:::: H^3HO
+
+## Reaction Arrows
+:::: A -> B
+:::: A <- B
+:::: A <-> B % not to be used according to IUPAC and ACS
+:::: A <--> B
```

Comment:
`rightleftarrows` seems little bit off on Chrome:
![image](https://user-images.githubusercontent.com/888148/48466088-21e82d00-e828-11e8-92b5-e52e50088d07.png)

Reply:
Yep. There was a problem with the SVG. Fixed now.